api/nakadi-event-bus-api.yaml

```diff
@@ -623,6 +623,50 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  '/event-types/{name}/cursors':
+      get:
+        tags:
+          - unmanaged-api
+          - monitoring
+          - management-api
+        security:
+          - oauth2: ['nakadi.event_stream.read']
+        description: |
+          Transforms a list of Cursors with `shift` into a list without `shift`s. This is useful when there is the need
+          for randomly access events in the stream.
+
+        parameters:
+          - name: name
+            in: path
+            description: EventType name
+            type: string
+            required: true
+          - name: X-Flow-Id
+            in: header
+            description: |
+              The flow id of the request, which is written into the logs and passed to called services. Helpful
+              for operational troubleshooting and log analysis.
+            type: string
+          - name: X-nakadi-cursors
+            in: header
+            description: |
+              Cursors indicating the positions to be calculated. If a cursors contains `shift` property, this property
+              will be used to calculate the actual offset for the given position and in the response the cursor will
+              no longer contain the `shift`.
```

Comment:
Is it possible to calculate this information for every possible cursor (for example if data related to new cursor was deleted)
If cursor will contain some invalid offset (for example less then -1 in case of kafka or value that is greater then latest offset), will it report some kind of error for this cursor?